projects/apache/cloudstack-cloudmonkey/Makefile

```diff
@@ -1,6 +1,6 @@
 BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
 GIT_TAG:=$(shell cat GIT_TAG)
-GOLANG_VERSION?="1.16"
+GOLANG_VERSION?="1.20"
```

Comment:
are they uysing this version upstream?

Reply:
Yes they released a new tag in April after 1.5 years. They jumped from 1.14 to 1.20 in their CI workflows.